### ISSUE TITLE
CafeTutorial_report_analysis. py Error

### ISSUE THREAD

**Issue by Bioinformatics-rookie** [hahnlab/CAFE#90]
When I ran CafeTutorial_report_analysis. py, I encountered an ImportError: No Module Named Cafecore. How to fix it

**Comment by gwct** [comment]
Hi,
Did you download the report analysis script from this page?
https://iu.app.box.com/v/cafetutorial-files/folder/22161236877
If so, you must also download the cafecore.py file and place it in the same
directory as the report analysis script.
Let us know if this still doesn't work!
-Gregg

**Comment by Bioinformatics-rookie** [comment]
Yes , I download from the page ,and I put the cafecore.py file into the same dir , it begain to work
Thanks

**Comment by Tiramisu023** [comment]
> https://iu.app.box.com/v/cafetutorial-files/folder/22161236877

The files are no longer valid.

**Comment by benfulton** [comment]
The files are now available again at https://github.com/hahnlab/cafe_tutorial .

**Comment by Tiramisu023** [comment]
> The files are now available again at https://github.com/hahnlab/cafe_tutorial .

Thanks for your work. Best wishes!
